src/App.js

```diff
@@ -0,0 +1,20 @@
+import Contact from "./components/Contact";
+import Footer from "./components/Footer";
+import Intro from "./components/Intro";
+import NavBar from "./components/NavBar";
+import Skills from "./components/Skills";
+import Works from "./components/Works";
+function App() {
+  return (
+    <div className="App">
+      <NavBar />
+      <Intro />
+      <Skills />
+      <Works />
+      <Contact />
+      <Footer />
+    </div>
+  );
+}
+
```

Comment:
- You may want to try implementing react router with the navigation and have some hands on experience, it is often a good way to demonstrate your understanding of the topic, what do you think?